migration-engine/core/src/api/error_rendering.rs

```diff
@@ -16,7 +15,7 @@ pub fn render_error(crate_error: CrateError) -> Error {
         })) => user_facing_error.into(),
         _ => UnknownError {
             message: format!("{}", crate_error),
-            backtrace: crate_error.backtrace().map(|bt| format!("{}", bt)),
+            backtrace: None,
```

Comment:
this is OK because we didn't produce any backtraces anyway. We can have backtraces again once they are stabilized in the standard library.